capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './baze_decorator/'
+
+class CapitalizeDecoratore
+  def correct_name
+    @nameable.correct_name.capitalize
+  end
+end
```

Comment:
- [ Required ] I think it would be better if the `CapitalizeDecoratore` inherits the characteristics of the `BaseDecorator` class for it to function correctly. Currently, any attempt to use the `@nameable` instance variable will lead to an error because the `@nameable` instance variable is not part of the `CapitalizeDecoratore` class. Please see the screenshot below and rectify this.

```ruby
require_relative 'base_decorator'

class CapitalizeDecoratore < BaseDecorator
  def correct_name
    @nameable.correct_name.capitalize
  end
end
```

## Error
![Screenshot from 2023-04-12 18-04-09](https://user-images.githubusercontent.com/16392046/231500177-62482bba-f062-473a-ba87-e19f588fe157.png)